lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -51,12 +51,6 @@
         description:
             - Name of the newly deployed guest
         required: True
-   name_match:
-        description:
-            - If multiple vms matching the name, use the first or last found
-        required: False
-        default: 'first'
-        choices: ['first', 'last']
```

Comment:
@bcoca This option was not released to the public yet. We do not have to be backward compatible with prior commits in devel, are we ?